linkerd/app/inbound/src/http/mod.rs

```diff
@@ -124,7 +124,11 @@ pub mod fuzz {
         I: io::AsyncRead + io::AsyncWrite + io::PeerAddr + Send + Unpin + 'static,
     {
         let connect = svc::stack(connect)
-            .push_map_target(|t: Http| Remote(ServerAddr(([127, 0, 0, 1], t.param()).into())))
+            .map_stack(|cfg, _, s| {
+                s.push_map_target(|p| Remote(ServerAddr(([127, 0, 0, 1], p).into())))
+                    .push_map_target(|t| Param::<u16>::param(&t))
+                    .push_connect_timeout(cfg.proxy.connect.timeout)
+            })
```

Comment:
is this change necessary? it seems to do the same thing afaict

Reply:
it's not necessary, the intent was just to ensure this code was in sync with the stack tests. i can back out this change.